MoltenVK/MoltenVK/GPUObjects/MVKImage.mm

```diff
@@ -1838,30 +1864,6 @@ static void signalAndUntrack(const MVKSwapchainSignaler& signaler) {
 			}
 			break;
 
-		case MTLPixelFormatDepth32Float_Stencil8:
```

Comment:
@billhollings I was hesitant to remove this for the same reason as the other comment above. Let me know if I missed something!

Reply:
Hmmm...now that you mention it, the code in this section might be about handling swizzles back in the old days before swizzling was available, where we just relied on a swap to a compatible format.

If possible, we should try to retain that for older systems.

Reply:
The check for the swizzle should probably be retained, especially for the case where `enableSwizzle()` is false. The format switch is handled by the newly refactored code above.